addons/l10n_es_pos/tests/test_frontend.py

```diff
@@ -37,3 +37,12 @@ def get_number_of_regular_invoices():
         num_of_regular_invoices = get_number_of_regular_invoices() - initial_number_of_regular_invoices
         self.assertEqual(num_of_simp_invoices, 3)
         self.assertEqual(num_of_regular_invoices, 1)
+
+    def test_pos_session(self):
+        simplified_partner = self.env.ref("l10n_es.partner_simplified", raise_if_not_found=False)
+        if simplified_partner:
+            simplified_partner.unlink()
+        self.assertFalse(self.env.ref("l10n_es.partner_simplified", raise_if_not_found=False))
+        self.main_pos_config.with_user(self.pos_user).open_ui()
+        self.start_tour("/pos/ui?config_id=%d" % self.main_pos_config.id, 'OpenSession', login="pos_user")
```

Comment:
I think you could try to create the order manually from the python test and avoid making a tour (also I think you forgot to add the tour file). Tour takes a lot of time to run so we should avoid them when it's possible

Reply:
> I think you could try to create the order manually from the python test and avoid making a tour (also I think you forgot to add the tour file). Tour takes a lot of time to run so we should avoid them when it's possible

The tour has been added for the open POS session. We don't need to create an order for this as an issue was occurring when opening the POS session.